Verslag/scope.tex

```diff
@@ -2,14 +2,32 @@ \section{Scope}
+During this analysis all sources that the applications connected to were collected. These sources could consist of IP-addresses and domain names. 
+The research allowed for the gathering of information of any potential servers with the use of tools such as shodan.io. 
+Connecting to these services however fell outside the scope of the research. This had the potential of raising unwanted attention of the malicious parties.
 
-For properly analyzing an Android application it was decided that doing a code analysis was very important, the code had potential to had revealed a lot of important information about what the application was doing that otherwise had been very hard to find. It had been important for the code analysis to get ahold of the source code of the software. This was achieved by using a decompiling program if the source of the application was not available.
+For the proper analysis of an Android application it was decided that doing a code analysis was essential, 
+the code had potential to had revealed a lot of important information about what the application was doing that otherwise had been very hard to find. 
+It had been important for the code analysis to get ahold of the source code of the software. 
+This was achieved by using a decompiling program if the source of the application was not available.
 
-For further understanding of the application, it had been decided that a process analysis was required. During this analysis, the memory and the processes that had been activated in the background were monitored. The different dependencies that the application used, had also been monitored.
+To further understand the application the decision has been made that a process analysis was required. 
+During this analysis, the memory and the processes that had been activated in the background were monitored. 
+The different dependencies that the application used, had also been monitored.
 
-During the research there was a possibility that a connection between a malicious application and a hidden database or server were found. This created a chance to potentially gain access to the server or database. It was decided that the actions within the research were within the confounds of the law. For this reason, it was crucial for the research that the international laws were being upheld. One of these laws had prohibited the research from interfering or tampering with the server or database without the express consent of the other party. Because of this reason, it was decided that there would be made notes about the connection and not interact with it in any way.
\ No newline at end of file
+During a reasearch like this there is a chance that the malicious application uses a command and control server. 
+This created a chance to potentially gain access to the server or database. It was decided that the actions within the research were within the confounds of the law. 
```

Comment:
It was decided that the actions within the research were within the confounds of the law.  Should be:

It was decided that the actions within the research should stay within the confounds of the law.